src/casp/services/api/routers/cell_operations_router.py

```diff
@@ -12,12 +13,15 @@
 
 
 @cell_operations_router.post(
-    "/annotate", response_model=t.Union[t.List[schemas.QueryCellDevDetails], t.List[schemas.QueryCell]]
+    "/annotate",
+    response_model=schemas.QueryAnnotationType,
```

Comment:
We talked offline but this should be a separate endpoint since it returns a different type of object